extensions/wikia/AdEngine/js/template/bfaaMobile.js

```diff
@@ -0,0 +1,101 @@
+/*global define, require*/
+define('ext.wikia.adEngine.template.bfaaMobile', [
+	'ext.wikia.adEngine.adHelper',
+	'ext.wikia.adEngine.context.uapContext',
+	'ext.wikia.adEngine.provider.btfBlocker',
+	'ext.wikia.adEngine.slotTweaker',
+	'ext.wikia.adEngine.video.uapVideoAd',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window',
+	require.optional('ext.wikia.adEngine.mobile.mercuryListener')
+], function (
+	adHelper,
+	uapContext,
+	btfBlocker,
+	slotTweaker,
+	uapVideoAd,
+	doc,
+	log,
+	win,
+	mercuryListener
+) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.template.bfaaMobile',
+		page,
+		unblockedSlots = [
+			'MOBILE_BOTTOM_LEADERBOARD',
+			'MOBILE_IN_CONTENT',
+			'MOBILE_PREFOOTER'
+		],
+		wrapper;
+
+	function runOnReady(iframe, params) {
+		var aspectRatio = params.aspectRatio,
+			adsModule = win.Mercury.Modules.Ads.getInstance(),
+			adjustPadding = function () {
+				var viewPortWidth = Math.max(doc.documentElement.clientWidth, win.innerWidth || 0),
+					height = aspectRatio ?
+						viewPortWidth / aspectRatio : iframe.contentWindow.document.body.offsetHeight;
+				page.style.paddingTop = height + 'px';
+				adsModule.setSiteHeadOffset(height);
+			},
+			onResize = adHelper.throttle(adjustPadding, 100);
+
+		page.classList.add('bfaa-template');
+		adjustPadding();
+
+		win.addEventListener('resize', onResize);
+		if (mercuryListener) {
+			mercuryListener.onPageChange(function () {
+				page.classList.remove('bfaa-template');
+				page.style.paddingTop = '';
+				adsModule.setSiteHeadOffset(0);
+				win.removeEventListener('resize', onResize);
+			});
+		}
+
+		if (params.videoUrl && params.videoTriggerElement) {
+			slotTweaker.onReady(params.slotName, function(iframe) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "iframe". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)